packages/apkid/.SRCINFO

```diff
@@ -0,0 +1,19 @@
+pkgbase = apkid
```

Comment:
`.SRCINFO`is use only for the AUR, we don't need that on BA repo.

Reply:
Oh sorry I thought they would be needed regardless, will delete it.

Reply:
You removed one but there is still `packages/apkid/.SRCINFO`